tests/flows/test_vesting.py

```diff
@@ -171,3 +172,143 @@ def test_main(
         web3=web3, address=get_address_of_type(config, "Staking")
     )
     assert side_staking.get_vesting_amount(erc20_token.address) == vesting_amount
+
+    # check if vesting is correct
+    available_vesting_before = side_staking.get_available_vesting(erc20_token.address)
+
+    # advance 100 blocks to see if available vesting increased
+    for _ in range(100):
+        # send 1 wei
+        send_ether(consumer_wallet, ZERO_ADDRESS, 1)
```

Comment:
do we have to send ether? can we send ocean instead? I think it is easier to mint fake ocean than fake ether (but I might be wrong)

Reply:
I would also update the comment to say something like "send dummy transactions to increase block count (not part of the actual functionality)"